src/components/navlink/NavLink.js

```diff
@@ -1,7 +1,11 @@
 import "./NavLink.css";
 
-function NavLink() {
-  return <a>Future Nav Icon</a>;
+function NavLink(props) {
+  return (
+    <a>
```

Comment:
Not sure if this should be an `a` element. Maybe better to use a button?

Reply:
Changed it.